digits/templates/models/images/classification/new.html

```diff
@@ -598,24 +614,83 @@ <h4 style="display:inline-block;">Python Layers</h4>
 
                             });
 
+
+function process_tf_vis(data){
+    // Processes a tensorflow graph_def string such that it is loaded in a tensorboard-style graph
+    var height = String($(window).height() - 200) + 'px' // Bootbox dynamic height hack
+    var the_data = '<script>function load() {document.getElementById("test1").pbtxt = ' + data + ' ;}</scr'+'ipt>';
+    the_data += '<link rel="import" href="{{ url_for('static', filename='tb/tf-graph-basic.build.html', ver=dir_hash) }}" onload=load()>';
+    the_data += '<div style="height:' + height + '"><tf-graph-basic id="test1"></tf-graph-basic></div>';
+    return the_data;
+}
+
+//copied from https://stackoverflow.com/questions/4565112/javascript-how-to-find-out-if-the-user-browser-is-chrome/13348618#13348618
```

Comment:
Let's use "From https://stackoverflow.com/a/13348618"

Similar to https://github.com/NVIDIA/DIGITS/blob/v5.0.0/digits/utils/__init__.py#L28